framework/utils/utils.py

```diff
@@ -15,9 +15,6 @@
   Utility module containing methods commonly used throughout the Python framework.
 """
 
-from __future__ import division, print_function, absolute_import
```

Comment:
This line is still needed because utils is can be called by plugin_installer.py which sometimes runs with python 2.

Reply:
I'll add it back in, with that comment as well. Thanks!